### PR TITLE
Made a header

### PR DESCRIPTION
**Pull request by LukeBacchus**

### CODE REVIEW

**Review by AlecFaye — APPROVED** [review]
Looks good.

**Comment by SachJay** [comment]
when you are adding dependencies make sure to add the --save flag to npm install <whatever> so it saves to our package.json then just commit that too so everyone has the new one

**Review by SachJay — CHANGES_REQUESTED** [review]
if you're adding dependencies make sure to add the --save flag to nom install and to commit the package.json and package.lock

File: my-app/src/App.js [hunk 1]
```diff
@@ -7,28 +7,30 @@ import {
...
-          </Route>
-          <Route path="/joingame">
-            <p>Replace this with join game component</p>
-          </Route>
-          <Route exact path="/beachspeech"> {/*may have to create sub routes depending on how we create the game*/}
-            <p>Replace this with game lobby component</p>
-          </Route>
-        </Switch>
-      </div>
+      <Header>
```
Comment (SachJay) [inline my-app/src/App.js]: We talked about this for a bit, but can we put it on the top figure out how to put it on the bottom if we really need a footer

File: my-app/src/App.js [hunk 2]
```diff
@@ -7,28 +7,30 @@ import {
...
+      <Switch> {/*you can change the endpoint name of your ticket to be more accurate to the page*/}
```
Comment (SachJay) [inline my-app/src/App.js:16]: Switch and Header on the same line, tab everything

File: my-app/src/components/Header/Header.css [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+    color: white !important;
+  }
+  
+  #header .MuiInputBase-input {
+    color: white !important;
+  }
+  
+  #header .MuiOutlinedInput-notchedOutline {
+    border-color: var(--light-purple) !important;
+  }
```
Comment (SachJay) [inline my-app/src/components/Header/Header.css]: git doesnt like it when there isnt a new line at the end of each file so can we just add it, same as Header.js

File: my-app/src/components/Header/Header.js [hunk 4]
```diff
@@ -0,0 +1,46 @@
...
+    {
+        label: 'Game Select',
+        path: '/gameselect'
+    },
+    {
+        label: 'Join Game',
+        path: '/joingame'
+    },
+    {
+        label: 'Beach Speech',
```
Comment (SachJay) [inline my-app/src/components/Header/Header.js]: we can remove this, should not be directly added

File: my-app/src/components/Header/Header.js [hunk 5]
```diff
@@ -0,0 +1,46 @@
...
+    }
+]
+
+class Header extends React.Component {
+    render() {
+        return (
+            <div id="header">
+                <div id="jumplist">
+                    {NAV_LINKS.map((navItem) => (
+                <Link
```
Comment (SachJay) [inline my-app/src/components/Header/Header.js]: tabs here too

File: my-app/src/components/Header/Header.js [hunk 6]
```diff
@@ -0,0 +1,46 @@
...
+        path: '/beachspeeech'
+    }
+]
+
+class Header extends React.Component {
+    render() {
+        return (
+            <div id="header">
+                <div id="jumplist">
+                    {NAV_LINKS.map((navItem) => (
```
Comment (SachJay) [inline my-app/src/components/Header/Header.js:27]: this whole thing is pretty cool

**Review by SachJay — CHANGES_REQUESTED** [review]
a few minor things but overall 👌

**Comment by SachJay** [comment]
your branch name is 'mainpage' but this ticket is about the header. Can you change the branch name to reflect thisss

**Pull request closed without merging**